Makefile

```diff
@@ -1,3 +1,8 @@
+PWD=$(shell pwd)
+export C_INCLUDE_PATH=${PWD}/extern/tmp
+export LIBRARY_PATH=${PWD}/extern/tmp
+export LD_LIBRARY_PATH=${PWD}/extern/tmp
+
```

Comment:
But the API is changed frequently in RocksDB. I hope they will only add new APIs later.